tests/FixerConfiguration/DeprecatedFixerOptionTest.php

```diff
@@ -128,11 +128,52 @@ public function testGetNormalizer(): void
     {
         $normalizer = static fn () => null;
 
-        $decoratedOption = $this->prophesize(FixerOptionInterface::class);
-        $decoratedOption->getNormalizer()->willReturn($normalizer);
+        $fixerOption = new class($normalizer) implements FixerOptionInterface {
```

Comment:
I needed to scroll so long that I forgot what was the test method about.
Maybe worth to have `$fixerOption = $this->getFixerOptionDouble()` approach, similar to other tests?